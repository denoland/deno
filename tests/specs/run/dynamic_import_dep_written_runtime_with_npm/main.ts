Deno.writeTextFileSync(
  "./b.ts",
  `
import { add } from "npm:@denotest/add";
console.log(add(1, 2));
`,
);

console.log("Loading...");
await import("./a.ts");
