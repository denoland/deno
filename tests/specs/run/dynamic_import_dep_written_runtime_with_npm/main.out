Loading...
Download http://localhost:4260/@denotest%2fadd
Download http://localhost:4260/@denotest/add/1.0.0.tgz
3
